pkg/controller/storagecluster/storageclasses.go

```diff
@@ -14,13 +14,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	// The following constants are the indices at which StorageClasses are returned from newStorageClasses and in
-	// which they should be passed to createStorageClasses.
-	cephFileSystemIndex = 0
-	cephBlockPoolIndex  = 1
-)
-
```

Comment:
I believe this block should be in the previous commit.

Reply:
@jarrpa, while the OBC index was removed in the previous commit, createSnapshotClasses in volumsnapshotterclasses.go still relies on these two index values in its switching block.

Reply:
Ah! I see. Thanks!